azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```
@@ -539,80 +853,97 @@ private void applyDisabledJobs(List<Object> disabledJobs,
 
   @Override
   public String submitExecutableFlow(ExecutableFlow exflow, String userId)
-      throws ExecutorManagerException {
+    throws ExecutorManagerException {
     synchronized (exflow) {
-      logger.info("Submitting execution flow " + exflow.getFlowId() + " by "
-          + userId);
-
-      int projectId = exflow.getProjectId();
       String flowId = exflow.getFlowId();
-      exflow.setSubmitUser(userId);
-      exflow.setSubmitTime(System.currentTimeMillis());
 
-      List<Integer> running = getRunningFlows(projectId, flowId);
-
-      ExecutionOptions options = exflow.getExecutionOptions();
-      if (options == null) {
-        options = new ExecutionOptions();
-      }
+      logger.info("Submitting execution flow " + flowId + " by " + userId);
 
       String message = "";
-      if (options.getDisabledJobs() != null) {
-        applyDisabledJobs(options.getDisabledJobs(), exflow);
-      }
+      if (queuedFlows.isFull()) {
+        message =
```

If it is a manual submit, how can we report this error back to user?